cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -107,14 +120,30 @@ func (mh *MessageHandle) HandleServer(container *mux.MessageContainer, writer mu
 		return
 	}
 
+	// handle the ack message from edge
+	if container.Message.Router.Operation == beehiveModel.ResponseOperation {
+		mutex, err := mh.getNodeLock(nodeID)
+		if err != nil {
+			klog.Errorf("Failed to get nodeLock, err: %v", err)
+			return
+		}
+
+		mutex.Lock()
```

Comment:
change `MessageAckChannel ` to `sync.Map` type?

Reply:
Done.